docs/doc.css

```diff
@@ -0,0 +1,190 @@
+@import url('https://fonts.googleapis.com/css2?family=Mochiy+Pop+P+One&family=Source+Sans+Pro:wght@200;400&family=Ubuntu&display=swap');
+
+.grey{
+    color:rgb(65, 63, 63);
+    width:fit-content;
+    margin-top:20px;
+}
+
+body{
+    font-family: 'Mochiy Pop P One', sans-serif;
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+    background-color: #f2f5f6;
+}
+nav{
+    color:white;
+    width:100%;
+    background-color:#084f70;
+    height:65px;
+    position: fixed;
+    z-index: 999;
+    top:0;
+}
+nav ul{
+    float:right;
+    margin-right:2rem;
+}
+.logo{
+    font-size:44px;
+    line-height:50px;
+    margin-left:60px;
+}
+.nav-links li{
+    font-size:24px;
+    display:inline-block;
+    line-height:60px;
+    padding-right:20px;
+    cursor: pointer;
+}
+.nav-links a{
+    color:white;
+    padding:10px;
+}
+ 
+.nav-links a:hover,.nav-links a:active{
+    background-color:#1b9bff;
+    padding:10px;
+}
+
+.side-drawer{
+    z-index:66;
+    background-color: #e4f1f7;
+    position:fixed;
+    width:220px;
+    height:100%;
+    left:0;
+    transition:all 0.5s ease; 
+    top:65px;
+}
+.side-drawer header{
+    font-weight: bold;
+   font-size:32px;
+   color:#12425a;
+   line-height:90px;
+   text-align:center;
+}
+.side-drawer a{
+    font-size:18px;
+    height:100%;
+    line-height:35px;
+    display:block;
+    color:white;
+    background-color: #084f70;
+    padding-left:40px;
+    box-sizing:border-box;
+    border-top:1px solid rgb(129, 230, 230);
+    border-bottom:2px solid rgb(182, 178, 178,.5);
+}
+.side-drawer a:hover{
+  padding-left:48px;
+}
+#check{
+    display:none;
+}
+label #btn,label #cancel{
+    position:absolute;
+    cursor:pointer;
+    background-color: #084f70;
+    border-radius:3px;
+}
+label #btn{
+    left:10px;
+    top:70px;
+    font-size:18px;
+    color:white;
+    padding:5px 9px;
+    transition: all .5 ease;
+}
+label #cancel{
+    z-index:999;
+    left:-200px;
+    top:60px;
+    font-size:26px;
+    color:#f9fcfd;
+    padding:2px 7px;
+    transition: all 0.5s ease;
+}
+#check:checked~.side-drawer{
+    left:0px;
+}
+#check:checked~label #btn{
+    left:250px;
+    opacity:0;
+    pointer-events: none;
+}
+#check:checked~label #cancel{
+  left:187px;
+}
+.text{
+    font-size:20px;
+}
+
+.Component-container{
+    margin-top:100px;
+    margin-left:400px;
+    background-color: #f2f5f6;
+}
+#frame{
+    left:0px;
+    margin-top:10px;
+    margin-bottom:10px;
+    margin-left:0px;
+    padding-left:0px;
+    width:70%;
+}
+.comp-title{
+    color:#084f70;
+}
+footer{
+    color:white;
+    margin-top:50px;
+    text-align:center;
+    line-height: 50px;
+    height:50px;
+    background-color: #084f70;
+    z-index: 7777;
```

Comment:
I dont think you will require that big of z-index. Try to keep it in range of 1-100.